torch/fx/experimental/recording.py

```diff
@@ -163,6 +163,13 @@ def is_evaluate_expr(self) -> bool:
     def is_defer_runtime_assert(self) -> bool:
         return self.name == "defer_runtime_assert"
 
+    def getarg(self, *, index: int, key: str) -> Any:
```

Comment:
What does this do? Can we get some docs?

Reply:
This is so we can get an argument from an event. It gets an argument by its positional index or its name.

Reply:
I guess we can get rid of it, though. It's not a good solution for getting arguments from events.